src/main/java/seedu/address/model/item/Event.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.model.item;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Stub class for the event class.
+ */
+public class Event {
+    private String date;
+
+    public String getDate() {
+        return this.date;
+    }
+
```

Comment:
date should be LocalDate?
missing setDate(Date)

Reply:
Do you think that LocalDateTime would be more suited instead?

Reply:
Yes! But I will leave the implementation to Bryan to decide what he would like to use.